deploy/kuberhealthy-prometheus.yaml

```diff
@@ -321,7 +321,7 @@ metadata:
   namespace: kuberhealthy
   annotations:
     prometheus.io/scrape: "true"
-    prometheus.io/port: "80"
+    prometheus.io/port: "8080"
```

Comment:
this will get regenerated by our build automation when the helm chart is updated - fyi